src/plugins/plot/src/configuration/XAxisModel.js

```diff
@@ -81,7 +81,8 @@ define([
                     min: bounds.start,
                     max: bounds.end
                 },
-                frozen: false
+                frozen: false,
+                regenerate: true
```

Comment:
All x-axes have this default property to force updating on screen tick text.

Reply:
I don't see this being set to false anywhere. Does this mean that x-axis ticks will be regenerated on every tick?

Reply:
@akhenry Yeah. This was a bad idea. Originally I didn't look at the code closely enough and thought updates were firing at the same interval. I was looking at scheduleTickUpdate which just ensures that multiple tick updates aren't fighting for their right... to plotly(?). I can see that ticks are updating too frequently with these changes.

I changed this to watch for an axis change on key. This works in respect to not firing unless needed.